src/programs/match_template/match_template.cpp

```diff
@@ -560,7 +574,6 @@ bool MatchTemplateApp::DoCalculation( ) {
             if ( ReturnThreadNumberOfCurrentThread( ) == 0 ) {
                 wxPrintf("Not rotating the search image for speed even though it is enabled\n");
             }
-            is_rotated_by_90 = false;
```

Comment:
Hard to see from the diff, but why is this no longer nescessary?

Reply:
It was always redundant. It is initialized to false.

Reply:
I think it is necessary because is_rotated_by_90 is still a class member. So when DoCalculate is called repeatedly by the gui, it might remain true even though the image has not been rotated.

Reply:
ah, you're right. This inheritance pattern is hard to remember. So MatchTemplatePanel is just collecting information and sending it to several instances of MatchTemplateApp and populating inherited job related information that MatchTemplateApp owns having derived from MyApp. MyApp is then handling several calls to the same instance of MatchTemplateApp::DoCalculation.

If that is true, then we should either a) never have member variables in programs run from the GUI or b) have myAPP reset state each time. I suppose (a) is much easier.

I'll get fix this and pixel_size.

Please add an corrections or details to my understanding. A simple flow chart to add to cistem-dev-docs explaining roughly how work is dispatched would be super useful!